diff --git a/node_modules/gaze-detection/index.js b/node_modules/gaze-detection/index.js
index 2cb62d3..2352f3e 100644
--- a/node_modules/gaze-detection/index.js
+++ b/node_modules/gaze-detection/index.js
@@ -4,7 +4,7 @@ import "@tensorflow/tfjs-backend-webgl";
 
 let model, video, rafID;
 let amountStraightEvents = 0;
-const VIDEO_SIZE = 500;
+const VIDEO_SIZE = 700;
 let positionXLeftIris;
 let positionYLeftIris;
 let event;
